package.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "pelias-admin-lookup",
-  "version": "1.0.1",
+  "version": "1.0.2",
```

Comment:
This isn't a big deal, but don't we usually up the version in a separate commit on master?